src/utils.py

```diff
@@ -60,6 +60,15 @@ def session(self):
 session = Postgres().session
 
 
+def format_date(date):
+    """Format date to a readable form."""
+    if date == 'N/A':
```

Comment:
you can rather check if date != "N/A": .....

Reply:
done